frontend/packages/vjs-plugin/src/styles.scss

```
@@ -154,6 +154,23 @@ $controlBarAnimationsDuration: 0.1s;
 	}
 }
 
+@keyframes onDisplayingDoubleTapSeekFeedback {
```

Stylelint (17.11.1)

[error] Expected keyframe name "onDisplayingDoubleTapSeekFeedback" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)